src/Fmt_odoc.mli

```diff
@@ -9,13 +9,14 @@
  *                                                                    *
  **********************************************************************)
 
-val fmt : Odoc_parser.Ast.docs -> Fmt.t
+val fmt :
+     Conf.t
```

Comment:
This argument is only used to call `fmt_code` so it should be removed and passed to `fmt_code` by calling code.